ansible/roles/h5p-staticfiles-server-core/templates/configmap.yml.j2

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: h5p-staticfiles-server-configmap
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: h5p-staticfiles-server
+data:  
+  NODE_ENV: "production"
+  CORS_ORIGIN: "http://h5p-staticfiles-server-svc:8080"
+  # Paths
+  SCHULCLOUD_BASE_PATH: "http://api-svc:3030/api/v1"
+  SCHULCLOUD_ALL_USERS_FOR_UUID: "http://api-svc:3030/api/v1/resolve/users/"
+  SCHULCLOUD_ALL_SCOPES_FOR_TOKEN: "http://api-svc:3030/api/v1/resolve/scopes/"
+  TZ: "Europe/Berlin"
+{% if CAL_IS_MIGRATION is defined %}
+  IS_MIGRATION: "{{ CAL_IS_MIGRATION }}"
+{% endif %}
```

Comment:
Where are this Enviorment variable's used at the running container?

Reply:
not at all on this moment deleted